pkg/gui/gui.go

```diff
@@ -246,6 +246,7 @@ func (gui *Gui) Run() error {
 		gui.goEvery(time.Millisecond*30, gui.reRenderMain)
 		gui.goEvery(dockerRefreshInterval, gui.refreshProject)
 		gui.goEvery(dockerRefreshInterval, gui.refreshContainersAndServices)
+		gui.goEvery(time.Millisecond*30000, gui.refreshImages)
```

Comment:
I would indeed make this configurable, defaulting to 30s

Reply:
Made the refresh time to be configurable